xbmc/cores/VideoPlayer/DVDInputStreams/InputStreamAddon.h

```diff
@@ -74,7 +74,9 @@ class CInputStreamAddon :
   virtual bool OpenDemux() override;
   virtual DemuxPacket* ReadDemux() override;
   virtual CDemuxStream* GetStream(int iStreamId) override;
+  virtual bool SupportsEnableAtPTS(){ return m_canEnableAtPTS; };
   virtual void EnableStream(int iStreamId, bool enable) override;
+  virtual void EnableStreamAtPTS(int iStreamId, uint64_t pts) override;
```

Comment:
It is DVD_TIME_BASE (microseconds). Addon devides it /1000 to come to miliseconds